.release-please-manifest.json

```
@@ -0,0 +1,3 @@
+{
+    ".": "5.78.0"
```

```suggestion
    ".": "5.78.0-preview"
```
Should this include the suffix too?